packages/jss-plugin-nested/src/index.js

```diff
@@ -94,15 +94,15 @@ export default function jssNested(): Plugin {
         // Replace all $refs.
         selector = selector.replace(refRegExp, replaceRef)
 
-        container.addRule(selector, style[prop], {...options, selector})
+        container.addRule(selector, style[prop], {...options, selector, safeReplace: true})
```

Comment:
proposal: lets make a new method - replaceRule which will do exactly this?

Reply:
you mean, we add relpaceRule in `StyleSheet`:
```
replaceRule(name: string, decl: JssStyle, options?: RuleOptions): Rule | null {
  if (options) {
    options.safeReplace = true
  }
  else {
    options = { safeReplace: true }
  }

  this.addRule(name, decl, options)
}
```
or just implement a same `addRule` method expect `safeReplace` logic ?

Reply:
> or just implement a same addRule method expect safeReplace logic ?

this, yeah

Reply:
it seems to me that it would be very obvious semantics

Reply:
But if we do that, which not add `safeReplace` option, we may have to add a `replace` method to `RuleList`. That adds a lot of  extra code.

Reply:
or maybe we can implement `addOrReplace` to share the same logic

Reply:
I will have a try

Reply:
I def. don't want to add a lot of extra code, it should share the logic